src/main/kotlin/com/terraformation/backend/seedbank/db/AccessionStore.kt

```diff
@@ -261,8 +261,7 @@ class AccessionStore(
                         .set(TOTAL_VIABILITY_PERCENT, accession.totalViabilityPercent)
                         .set(TREES_COLLECTED_FROM, accession.numberOfTrees)
                         .returning(ID)
-                        .fetchOne()
-                        ?.get(ID)!!
+                        .fetchOne(ID)!!
```

Comment:
previous change `fetchOne()?.get(ID)!!` is more forgiving than `fetchOne(ID)!!` ?
Guessing fetchOne will always yield a row and the `?` was not needed.

Reply:
The behavior should be the same. Old version:

1. `fetchOne()` returns null if no row matches
2. `?.get(ID)` passes the null through (evaluates to null)
3. `!!` operator throws an exception on the null

New version:

1. `fetchOne(ID)` returns null if no row matches
2. `!!` operator throws an exception on the null

The old version could have been written `fetchOne()!!.get(ID)!!` instead.